coordinate_system_name: "user_coordinate_system"
coordinate_system_resolution: 574
coordinate_system_distance: 3000 

# Projector Reference System {P}
P1: {x:  10, y:  10}
P2: {x: 584, y:  10}
P3: {x: 584, y: 410}
P4: {x:  10, y: 410}

# User Reference System {T}
T1: {x: 10, y: 10}

# T2, T3, T4 will be autodefined with the resolution. See the docs
#      ____________________      
#   T4|                    |T3   ^
#     |                    |     | 
#     |                    |     | autoscale * resolution
#     |                    |     |
#   T1|____________________|T2   v
#      <--  autoscale * --> 
#           resolution

#  If {P} defines a square, autoscale = 1
#  If {P} defines a rectangle, autoscale = 1 only for largest dimension. 